Terminal.Gui/Core/View.cs

```diff
@@ -3155,5 +3158,45 @@ public View GetTopSuperView (View view = null, View superview = null)
 
 			return top;
 		}
+
+		/// <summary>
+		/// Finds which view that belong to the <paramref name="start"/> at the provided location.
+		/// </summary>
+		/// <param name="start">The view where to look for.</param>
+		/// <param name="x">The column location.</param>
+		/// <param name="y">The row location.</param>
+		/// <param name="resx">The found view column location.</param>
+		/// <param name="resy">The found view row location.</param>
+		/// <returns>The view that belong to the provided location.</returns>
```

Comment:
"The view that was found at the <praramref name"x"/> and <praramref name"x"/> coordinates. `null` if no view was found."